src/connectors/aws/index.ts

```diff
@@ -196,17 +194,19 @@ export class AWSService {
 
   // no-op if sqs not initialized; when env sqs queue-url not set
   sqsSendMessage = async ({
-    MessageGroupId,
-    MessageBody,
+    messageBody,
+    queueUrl,
+    messageGroupId,
   }: {
-    MessageGroupId: string
-    MessageBody: any
+    messageBody: any
+    queueUrl: string
```

Comment:
how about it's an enum type instead of passing a URL directly?

Reply:
good idea~